src/View/Cell.php

```diff
@@ -259,15 +260,17 @@ protected function _cacheConfig($action)
      * This is because PHP will not allow a __toString() method to throw an exception.
      *
      * @return string Rendered cell
+     * @throws \Error Include error details for PHP 7 fatal errors.
      */
     public function __toString()
     {
         try {
             return $this->render();
         } catch (Exception $e) {
             trigger_error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()), E_USER_WARNING);
-
             return '';
+        } catch (Error $e) {
+            throw new Error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()));
```

Comment:
@thinkingmedia I don't think this is a good idea. It would be better to cast return value of `render()` to string instead:
```php
public function __toString()
{
     try {
         return (string)$this->render();
     } catch (Exception $e) {
         trigger_error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()), E_USER_WARNING);
         return '';
     }
}
```
 `render()` seems to return null when error occurs. Isn't that why you have gotten a fatal error?